client/signup/steps/import-preview/index.jsx

```diff
@@ -0,0 +1,209 @@
+				],
+				unsupported: translate(
+					"We're unable to import your theme, plugins, comments, or online store."
+				),
+			},
+			'godaddy-gocentral': {
+				supported: [
+					translate( 'Pages' ),
+					translate( 'Blog posts' ),
+					translate( 'Images' ),
+					translate( 'Other media' ),
+				],
+				unsupported: translate(
+					"We're unable to import your online store, site layout, or styling."
+				),
+			},
+			medium: {
+				supported: [
+					translate( 'Posts' ),
+					translate( 'Images' ),
+					translate( 'Other media' ),
+					translate( 'Tags' ),
+				],
+				unsupported: translate( "We're unable to import your drafts, comments or followers." ),
+			},
+			squarespace: {
+				supported: [
+					translate( 'Posts' ),
+					translate( 'Pages' ),
+					translate( 'Images' ),
+					translate( 'Comments' ),
+					translate( 'Tags' ),
+				],
+				unsupported: translate(
+					"We're unable to import your online store, template, layout or styling."
+				),
+			},
+			wix: {
+				supported: [
+					translate( 'Pages' ),
+					translate( 'Blog posts' ),
+					translate( 'Images' ),
+					translate( 'Other media' ),
+				],
+				unsupported: translate(
+					"We're unable to import your online store, site layout, or styling."
+				),
+			},
+		};
+	}
+
+	componentDidMount() {
+		this.props.saveSignupStep( { stepName: this.props.stepName } );
+	}
+
+	submitImportPreview() {
+		this.props.submitSignupStep( { stepName: this.props.stepName } );
+		this.props.goToNextStep();
+	}
+
+	renderIcon() {
+		const {
+			signupDependencies: { importSiteEngine },
+		} = this.props;
+
+		const importerData = importerConfig()[ importSiteEngine ];
+
+		return (
+			<ImporterLogo
+				icon={ importerData.icon }
+				className={ 'import-preview__icon importer__service-icon ' + importSiteEngine }
+			/>
+		);
+	}
+
+	renderSupported() {
+		const {
+			signupDependencies: { importSiteEngine },
+		} = this.props;
+
+		// The preceding steps always provide a value for importSiteEngine
+		const items = this.previewData[ importSiteEngine ].supported || [];
+
+		const listItems = items.map( item => {
+			return (
+				<li className="import-preview__supported-item" key={ item }>
+					<Gridicon icon="checkmark" size={ 18 } className="import-preview__checkmark" />
+					<div className="import-preview__supported-text">{ item }</div>
+				</li>
+			);
+		} );
+
+		return <ul className="import-preview__supported">{ listItems }</ul>;
+	}
+
+	renderPreview() {
+		const {
+			signupDependencies: { importSiteEngine, importSiteUrl, siteTitle },
+			translate,
+		} = this.props;
+
+		let cardTitle = siteTitle;
+		let trimmedSiteUrl = '';
+
+		if ( isEmpty( importSiteUrl ) ) {
+			// The user has an import file and told us the source
+			const importerData = importerConfig()[ importSiteEngine ];
+			cardTitle = translate( 'Importing from ' ) + importerData.title;
+		} else {
+			trimmedSiteUrl = importSiteUrl.replace( /^https?:\/\//i, '' ).replace( /\/$/, '' );
```

Comment:
Too tired to make a combined regex pattern work right now, so I've done a double `replace`.

Reply:
If you want to switch it, this should work, but I think it's also fine as two replaces:
```suggestion
			trimmedSiteUrl = importSiteUrl.replace( /^https?:\/\/(.*?)\/$/, '$1' );
```

Reply:
I found I had to make the outer subpatterns optional, but now it works, thanks Jon.